### DISCUSSION TITLE
Self-Hosted: On-Demand checkbox via Dependency Dashboard missing.

### DISCUSSION THREAD

**Discussion by Zayik** [renovatebot/renovate#22544] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.91.1

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

When using a self-hosted Renovate, when it adds the Dependency Dashboard, I would like the " Check this box to trigger a request for Renovate to run again on this repository" checkbox to exist. 

Additionally, if the feature does exist, I would like some documentation to be created that indicates how to add this feature for the self-hosted runner as all documentation seems to indicate it should always exist. 

### What you tried so far.

I've scoured through the documentation regarding the Dependency Dashboard and I didn't find much. No indication that its something that is toggleable. 
I've inspected the html code to see if the sections were created but just invisible. They did not exist. 

I've tried to add in the dependencyDashboard enabled flag in the renovate.json being used. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Duplicate of #22523
